Documentation/ApiOverview/Database/DatabaseStructure/Index.rst

```diff
@@ -62,9 +63,10 @@ There are certain requirements for such managed tables:
    consistency it is recommended to name them that way.
 
 
+.. index:: Tables; Pages
```

Comment:
This will probably end up irritating. Please consider writing `Tables; 'pages'` or `Tables; "pages"`. Otherwise, following the logic, we would have `Tables; Tt_content` somewhere.